dockerfiles/che/entrypoint.sh

```diff
@@ -291,7 +291,7 @@ init() {
 }
 
 add_cert_to_truststore() {
-  DEFAULT_JAVA_TRUST_STORE=$JAVA_HOME/lib/security/cacerts
+  DEFAULT_JAVA_TRUST_STORE=${JAVA_TRUST_STORE:-${JAVA_HOME}/lib/security/cacerts}
```

Comment:
Is `JAVA_TRUST_STORE` an standart env property? Or how would we know we can set it ?

Reply:
should we document that somehow?

Reply:
isn't it already common that you have variables called DEFAULT_SOMETHING and SOMETHING overrides it?

Anyway https://github.com/redhat-developer/codeready-workspaces/pull/325/files#diff-3254677a7917c6c01f55212f86c57fbfR23 is how you would use this but if you don't set it you get the exact same behaviour you have today where you just force it to to be `$JAVA_HOME/lib/security/cacerts`

Reply:
Note that you are explicitly setting `JAVA_TRUST_STORE=/home/user/cacerts` on line 297/296 below. 

https://github.com/eclipse/che/pull/17465/files#diff-1da4013f60e1b92b5613cf9680423a88R296

So maybe the logic here doesn't make sense ... I can probably set up symlinks in downstream CRW build so that `${JAVA_HOME}/lib/security/cacerts` is a valid path.